### PR TITLE
Add validation for business registration model

### PR DESCRIPTION
**Pull request by MartynaUb**

### CODE REVIEW

File: Support_Your_Locals/Startup.cs [hunk 1]
```diff
@@ -34,7 +34,6 @@ public void ConfigureServices(IServiceCollection services)
                 InitialCatalog = Configuration["Database:Name"],
                 UserID = Configuration["Database:Username"],
                 Password = Configuration["Database:Password"],
-                Encrypt = true,
```
Comment (EvalVis) [inline Support_Your_Locals/Startup.cs]: Why did you remove this?

File: Support_Your_Locals/Views/Business/AddAdvertisement.cshtml [hunk 2]
```diff
@@ -70,14 +72,17 @@
                     <div class="form-group">
                         <label asp-for="Header">Header</label>
                         <input asp-for="Header" class="form-control">
+                        <span asp-validation-for="Header"></span>
```
Comment (EvalVis) [inline Support_Your_Locals/Views/Business/AddAdvertisement.cshtml:77]: You don't need to hardcode this. Just use     `<div` class="text-danger" `asp-validation-summary="All"></div>`
This will act as `<span` `asp-validation-for="Header"></span>` and `<span` `asp-validation-for="Description"></span>` and so on.

File: Support_Your_Locals/Models/ViewModels/BusinessRegisterModel.cs [hunk 3]
```diff
@@ -6,17 +6,17 @@ namespace Support_Your_Locals.Models.ViewModels
...
         public string PhoneNumber {get; set;}
-        [Required(ErrorMessage = "Please enter your business header")]
+        [Required(ErrorMessage = "Please enter header for your business")]
         public string Header {get; set;}
         public string Longitude { get; set; }
         public string Latitude { get; set; }
         public string Picture { get; set; }
-        public TimeSheetRegisterViewModel[] Workdays { get; set; } = new TimeSheetRegisterViewModel[7];
-        public List<Product> Products { get; set; } = new List<Product>();
+        public List<TimeSheetRegisterViewModel> Workdays { get; set; } = new List<TimeSheetRegisterViewModel>();
```
Comment (EvalVis) [inline Support_Your_Locals/Models/ViewModels/BusinessRegisterModel.cs]: This should be array of size 7. If not only edit will work (empty list will not be rendered on add advertisement). I can not check it exactly because I do not have access to a database now.

File: Support_Your_Locals/Startup.cs (hunk 1 above)
Comment (MartynaUb) [reply Support_Your_Locals/Startup.cs]: Encrypt was removed because In memory database does not support it and I wanted to use in local memory db when working on new features. 
We could have this setting configurable if we want to have it on the azure but I am not sure if we want to encrypt data at all. I understand that encryption of the database data is useful when storing user sensitive data that needs to follow GDPR and while our app is not used with real data we can skip this. What do you think?

Comment (EvalVis) [reply Support_Your_Locals/Startup.cs]: Agreed.

**Review by Yar3k — APPROVED** [review]

File: Support_Your_Locals/Models/ViewModels/BusinessRegisterModel.cs [hunk 4]
```diff
@@ -6,13 +6,14 @@ namespace Support_Your_Locals.Models.ViewModels
...
     {
 
+
         public long BusinessId { get; set; } //Danger zone.
-        [Required(ErrorMessage = "Please add your business description")]
+        [Required(ErrorMessage = "Please add description for your business")]
         public string Description {get; set;}
-        [RegularExpression(@"^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$", ErrorMessage = "Bad phone number")]
-        [Required(ErrorMessage = "Please enter your business phone number")]
+        [RegularExpression(@"^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$", ErrorMessage = "Entered phone number is not correct")]
```
Comment (EvalVis) [inline Support_Your_Locals/Models/ViewModels/BusinessRegisterModel.cs:13]: Should not data annotation `[Phone]` be enough?

**Review by EvalVis — APPROVED** [review]
I have some questions you haven't responded. However, you have my approval.
